core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceSettingServiceUnitTests.scala

```diff
@@ -434,8 +439,12 @@ class WorkspaceSettingServiceUnitTests extends AnyFlatSpec with MockitoTestUtils
       LifecycleAction.newDeleteAction(),
       LifecycleCondition.newBuilder().setMatchesPrefix(List("muchBetterPrefix").asJava).setAge(31).build()
     )
-    when(gcsDAO.setBucketLifecycle(workspace.bucketName, List(newSettingGoogleRule)))
-      .thenReturn(Future.failed(new Exception("failed to apply settings")))
+    when(
+      gcsDAO.setBucketLifecycle(ArgumentMatchers.eq(workspace.bucketName),
+                                ArgumentMatchers.eq(List(newSettingGoogleRule)),
+                                any()
+      )
+    ).thenReturn(Future.failed(new Exception("failed to apply settings")))
 
     val service =
```

Comment:
There is no unit test coverage of calling the `gcsDAO` for soft delete or requester pays settings?